packages/backend/package.json

```diff
@@ -49,7 +49,7 @@
     "@as-integrations/koa": "^1.1.1",
     "@faker-js/faker": "^8.0.2",
     "@interledger/http-signature-utils": "1.0.3",
-    "@interledger/open-payments": "1.1.0",
+    "@interledger/open-payments": "2.0.0",
```

Comment:
Do we need to change anything in the remote incoming payment service, when we use the open payments client?

Reply:
Not that I can see. The field usage was already updated (using `metadata`, not using `externalRef`/`description`) in `deps.openPaymentsClient.incomingPayment.create`. Beyond that, I confirmed the open payments client is created from the `createOpenPaymentsClient` function imported from `2.0.0`. The other methods on the client used in the service (grants, paymentPointer, token) weren't changed. Any other concerns or scenarios I should test?

In digging around I did notice the auth package needs to be updated as well.